src/components/layout/RightSection.vue

```diff
@@ -1,44 +1,69 @@
 <template>
   <div class="app__rightcolumn">
     <div class="content">
-      <div class="section wallet-install" :class="{ active: !isLoggedIn }" v-if="!loading.wallet">
+      <div
+        v-if="!loading.wallet"
+        class="section wallet-install"
+        :class="{ active: !isLoggedIn }"
+      >
         <div class="section__title">
-          <img src="../../assets/iconWallet.svg" />
+          <img src="../../assets/iconWallet.svg">
           Wallet
         </div>
         <div class="section__body clearfix">
-          <div><a :href="downloadUrl" target="_blank" class="button w-100">Install Wallet</a></div>
+          <div>
+            <a
+              :href="downloadUrl"
+              target="_blank"
+              class="button w-100"
+            >
+              Install Wallet
+            </a>
+          </div>
         </div>
       </div>
       <div class="section trending">
         <div class="section__title">
-          <img src="../../assets/iconTrending.svg" />
+          <img src="../../assets/iconTrending.svg">
           Trending
         </div>
         <div
+          v-if="!loading.tips"
           class="section__body topics-section"
           :class="{ active: topics.length > 0 }"
-          v-if="!loading.tips"
         >
-          <div class="section__item" v-for="([topic, data], idx) in topics" :key="idx">
+          <div
+            v-for="([topic, data], idx) in topics"
+            :key="idx"
+            class="section__item"
+          >
             <div class="topic-container text-ellipsis">
               <topic :topic="topic" />
             </div>
             <div>
               <span class="value">
-                <ae-amount :amount="data.amount" :round="2"></ae-amount>
+                <ae-amount
+                  :amount="data.amount"
+                  :round="2"
+                />
               </span>
-              <fiat-value :amount="data.amount"></fiat-value>
+              <fiat-value :amount="data.amount" />
```

Comment:
is this required by the linter?
it's kind of contradicts
https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended

Reply:
It is not, in this link it is explained very clearly. We don't have a DOM template that is written in html file. We have a single-file components that recommended to close like that <MyComponent/> if there is no content.